src/Grpc.Core.Api/.gitignore

```diff
@@ -0,0 +1,2 @@
+bin
```

Comment:
root .gitignore should have these already

Reply:
removed.